src/main/java/org/myrobotlab/service/config/ServoConfig.java

```diff
@@ -0,0 +1,22 @@
+package org.myrobotlab.service.config;
+
+public class ServoConfig extends Config {
+
+  public Boolean autoDisable;
+  public String controller;
+  public Double currentOutputPos;
```

Comment:
we can specify the current input postion. .we should not put output current position in the config.